templates/person/show.html.twig

```diff
@@ -166,9 +166,9 @@
 {% endblock %}
 
 {% block javascripts %}
-    <script src="{{ asset('yarn/tablesorter/dist/js/jquery.tablesorter.min.js') }}"></script>
+    <script src="{{ asset('node_modules/tablesorter/dist/js/jquery.tablesorter.min.js') }}"></script>
     {% if makeTimeline %}
-        <script src="{{ asset('yarn/@knight-lab/timelinejs/dist/js/timeline-min.js') }}"></script>
+        <script src="{{ asset('node_modules/@knight-lab/timelinejs/dist/js/timeline-min.js') }}"></script>
```

Comment:
Can you change this path to `node_modules/@knight-lab/timelinejs/dist/js/timeline.js` (it looks like they removed the `-min` and just made the minified one the default)

Reply:
Fixed